### ISSUE TITLE
[Accessibility]: Search notebooks color contrast poor for link in code block in markdown

### ISSUE THREAD

**Issue by rvantonder** [sourcegraph/sourcegraph#35164]
### Audit type

ARC Toolkit

### User journey audit issue

Notebooks

### Problem description

Title

### Expected behavior

Better contrast

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by rvantonder** [comment]
https://sourcegraph.com/notebooks/Tm90ZWJvb2s6MTAwNw==
![Screen Shot 2022-05-18 at 6 44 17 PM](https://user-images.githubusercontent.com/888624/169185939-c3e213f8-db20-4580-9957-2fae7ad5dec8.png)
